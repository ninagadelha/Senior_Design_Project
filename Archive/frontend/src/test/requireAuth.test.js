import renderer from "react-test-renderer";
import RequireAuth from "../routes/requireAuth";

jest.mock("react-router-dom");

const renderTree = (tree) => renderer.create(tree);
describe("<RequireAuth>", () => {
  it("should render component", () => {
    expect(renderTree(<RequireAuth />).toJSON()).toMatchSnapshot();
  });
});
