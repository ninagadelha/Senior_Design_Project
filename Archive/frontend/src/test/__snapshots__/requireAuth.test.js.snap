// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RequireAuth> should render component 1`] = `null`;
